Documentation/media-migration.md

```diff
@@ -1,151 +1,278 @@
+
+### Buffer and Seek APIs
+| Media SDK | AEPRoku SDK|
+| -- | -- |
+| `mediaTrackEvent(ADBMobile().MEDIA_BUFFER_START, bufferInfo, bufferContextData)` | `sendMediaEvent(bufferCompleteXDM)` |
+| `mediaTrackEvent(ADBMobile().MEDIA_BUFFER_COMPLETE, invalid, invalid)` | `sendMediaEvent(bufferCompleteXDM)` |
+| `mediaTrackEvent(ADBMobile().MEDIA_SEEK_START, seekInfo, seekContextData)` | `sendMediaEvent(pauseStartXDM)` |
+| `ADBMobile().mediaTrackEvent(ADBMobile().MEDIA_SEEK_COMPLETE, invalid, invalid)` | `sendMediaEvent(pauseStartXDM)` |
+
+> [!NOTE]
+> For tracking seek in AEPRoku SDK, use eventType `pauseStart` with correct playhead. Media backend will detect seek based on playhead and timestamp values.
+
+### Chapter APIs
+| Media SDK | AEPRoku SDK|
+| -- | -- |
+| `mediaTrackEvent(ADBMobile().MEDIA_CHAPTER_START, chapterInfo, chapterContextData)` | `sendMediaEvent(chapterStartXDM)` |
+| `mediaTrackEvent(ADBMobile().MEDIA_CHAPTER_COMPLETE, invalid, invalid)` | `sendMediaEvent(chapterCompleteXDM)` |
+| `mediaTrackEvent(ADBMobile().MEDIA_CHAPTER_SKIP, invalid, invalid)` | `sendMediaEvent(chapterSkipXDM)` |
+
+### Quality of Experience and Error APIs
+| Media SDK | AEPRoku SDK|
+| -- | -- |
+| `mediaUpdateQoS(qosinfo)` | NA |
+| `mediaTrackEvent(ADBMobile().MEDIA_BITRATE_CHANGE)` | `sendMediaEvent(pingXDM)` |
+| `mediaTrackError(errorId, ADBMobile().ERROR_SOURCE_PLAYER)` | `sendMediaEvent(errorXDM)` |
+
+> [!NOTE]
+> QoE info can be attached to any event's xdm data the sendMediaEvent(eventXDM) API. Refer to the [QoeDataDetails](https://github.com/adobe/xdm/blob/master/docs/reference/datatypes/qoedatadetails.schema.md) fieldgroup
+
+### Playhead update API
+| Media SDK | AEPRoku SDK|
+| -- | -- |
+| `mediaUpdatePlayhead(position)` | NA (sent with all the APIs in the xdm data) |
+
+## PlayerState Tracking API
+| Media SDK | AEPRoku SDK|
+| -- | -- |
+| NA | `sendMediaEvent(statesUpdateXDM)` |
+
+### Helper APIs
+| Media SDK | AEPRoku SDK|
+| -- | -- |
+| `adb_media_init_mediainfo(title, id, length, streamType, mediaType)` | NA |
+| `adb_media_init_adbreakinfo(title, startTime, position)` | NA |
+| `adb_media_init_adinfo(title, id, position, duration)` | NA |
+| `adb_media_init_chapterinfo(title, position, length, startTime)` | NA |
+| `adb_media_init_qosinfo(bitrate, startupTime, fps, droppedFrames)` | NA |
+
+## Initliaze SDK instance
+
+**Media SDK**
+
+```brightscript
+' Create adbmobileTask node
+m.adbmobileTask = createObject("roSGNode","adbmobileTask") <br> Get AdobeMobile SG connector instance
+' Create SDK instance
+m.adbmobile = ADBMobile().getADBMobileConnectorInstance(m.adbmobileTask)
+```
 
-If you are currently using the Media Analytics SDK for your Roku projects, this guide offers a detailed outline on how to migrate to AEP Roku SDK using a step-by-step approach.
+**AEPRoku SDK**
 
-In AEP Roku SDK, there are two Media trakcing APIs:
+```brightscript
+' Create SDK instance
+m.aepSdk = AdobeAEPSDKInit()
+```
```

Comment:
Can we add a link to the `getting started` doc?

Reply:
Should we add it in prereq?

Reply:
Added in prereq. Let me know if your thoughts.

Reply:
Yup, it works.